rts/Lua/LuaVAO.cpp

```diff
@@ -74,6 +73,15 @@ LuaVAOs::~LuaVAOs()
 	luaVAOs.clear();
 }
 
+
+/***
+ *
+ * @function gl.GetVAO
+ * @treturn nil|VAO the VAO ref on success, else nil
+ * @usage
+ * local myVAO = gl.GetVAO()
+ * if myVAO == nil then Spring.Echo(“Failed to get VAO”) end
```

Comment:
I don't think these quotation marks actually work? Would be good if the code was copy-pastable.
```suggestion
 * if myVAO == nil then Spring.Echo("Failed to get VAO") end
```

Reply:
Copied from gdocs, thank you! Fixed on VBO too